docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -42,42 +42,44 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
 
 ### Parameters
 
+* **grouping_expression**
+
+    Specifies the criteria based on which the rows are grouped together. The grouping of rows is performed based on
+    result values of the grouping expressions. A grouping expression may be a column name like `GROUP BY a`, a column position like
+    `GROUP BY 0`, or an expression like `GROUP BY a + b`.
+
 * **GROUPING SETS**
 
     Groups the rows for each subset of the expressions specified in the grouping sets. For example,
```

Comment:
`Groups the rows for each grouping set specified after GROUPING SETS. For example, ...`

Reply:
Done